eng/versioning/alternative_external_dependency_versions.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Not a fan of the file name `alternative_external_dependency_versions`, thoughts on something more descriptive to what this represents such as `supported_external_dependency_versions` where it's clear to both us as the repo maintainers and to someone external that these are dependency versions we support. Possibly add a qualifier to this for Track 2 or data plane if this isn't supported by management plane (and it's not supported by Track 1)